ci/playbooks/tcib/tcib.yml

```diff
@@ -15,6 +15,15 @@
   hosts: "{{ cifmw_target_host | default('localhost') }}"
   gather_facts: true
   tasks:
+    - name: Copy repo-setup generated repos to yum.repos.d directory
```

Comment:
actually you may just push a new parameter to the job, `cifmw_repo_setup_output`, to get the repos in the right location.

Reply:
We already have a repo-setup output directory var [`cifmw_repo_setup_output`](https://github.com/openstack-k8s-operators/ci-framework/blob/main/ci_framework/roles/repo_setup/defaults/main.yml#L31) which is defaulted to [`cifmw_build_containers_repo_dir`](https://github.com/openstack-k8s-operators/ci-framework/blob/main/scenarios/centos-9/tcib.yml#L7). So we do not need that.

These repos are mounted with in the container during container build process. On the host system, there might be other repos in /etc/yum.repos.d directory. If we generate repos directly in yum.repos.d directory and mount it with container build process. Then we might include additional repos. it will affect the testing. That's why i have added this task to specifically copy the repos to yum.repos.d directory on host system to install python3-tcib rpm.

Reply:
fair enough